scripts/circle-ci/run-circle-tests.sh

```diff
@@ -9,7 +9,7 @@ function checkDocsBuild {
   make html
 }
 
-CONTAINER_1=(':atlasdb-cassandra-integration-tests:check')
+CONTAINER_1=(':atlasdb-cassandra-integration-tests:check :atlasdb-jepsen-tests:check')
```

Comment:
I'm not sure I understand this part 100%, but do we need Timelock Server to merge before this can run on circle? It looks like we currently depend on the ETE project, which _will_ have Timelock Server at some point, but doesn't have it yet.